src/configure/helper/azureSessionHelper.ts

```diff
@@ -0,0 +1,14 @@
+import { extensionVariables, SubscriptionSession } from "../model/models";
+
+export function getSubscriptionSession(subscriptionId: string): SubscriptionSession {
```

Comment:
should this return subscription.session, which corresponds to its name.

Reply:
you are right, I can directly return .session. When I started I thought of using both session and subscription and hence was returning SubscriptionSession. But it is no longer required. Will return session directly and remove SubcriptionSession definition from model as well